src/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>leaderboard</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />

```

Comment:
- Hey, let's kindly update our style by aligning the refresh button correctly to match the design template given below:

*wireframe*
![Screenshot from 2023-02-08 12-40-22](https://user-images.githubusercontent.com/59397159/217522370-7e9ed387-6d0a-4bee-9f68-635fe91932ba.png)

*your design*
![Screenshot from 2023-02-08 12-37-31](https://user-images.githubusercontent.com/59397159/217522381-aab2a331-49c8-46af-9853-82bc8c1ecc5d.png)